src/librustc/lib.rs

```diff
@@ -50,7 +50,6 @@
 #![feature(from_ref)]
 #![feature(fs_read_write)]
 #![cfg_attr(windows, feature(libc))]
-#![feature(macro_lifetime_matcher)]
```

Comment:
You still need `#![cfg_attr(stage0, feature(macro_lifetime_matcher))]` for these.